src/chatlog/chatmessage.cpp

```diff
@@ -249,7 +249,7 @@ QString ChatMessage::detectAnchors(const QString &str)
                 "(?:\\b)((www\\.)|(http[s]?|ftp)://)" // (protocol)://(printable - non-special character)
                 // http://ONEORMOREALHPA-DIGIT
                 "\\w+\\S+)" // any other character, lets domains and other
-                "|(?:\\b)(file:///)([\\S| ]*)" //link to a local file, valid until the end of the line
+                "|(?:\\b)((file|smb)://)([\\S| ]*)" //link to a local file, valid until the end of the line
```

Comment:
could you correct the comment?

I.e. with `file://` or `smb://` it's no longer about only local files.